src/Parser/Parser.hs

```diff
@@ -217,6 +216,19 @@ reservedOp = P.reservedOp lexer
 comma :: ParsecT String u Identity String
 comma = P.comma lexer
 
+-- | A parser for a literal expression
+literal :: Parser (Expr SourcePos)
```

Comment:
If we decide that only literals should be allowed, we can use this

Reply:
Agreed